test/ownership/Whitelist.test.js

```diff
@@ -27,7 +27,7 @@ contract('Whitelist', function (accounts) {
     it('should add address to the whitelist', async function () {
       await expectEvent.inTransaction(
         mock.addAddressToWhitelist(whitelistedAddress1, { from: owner }),
-        'WhitelistedAddressAdded'
+        'RoleAdded'
```

Comment:
Ahh, I see, we were triggering two events.
It would be nice to check here that the role added is `whitelist`.